app/importers/market_research_data.rb

```diff
@@ -3,9 +3,9 @@
 
 class MarketResearchData
   include Importable
-  include VersionableResource
```

Comment:
@calumhalcrow, if the source is no longer going to be updating, won't doing the version check make it so the importer never runs?  We still need it to catch Industry Mapper updates.